client/src/core/RuntimeTypes.res

```diff
@@ -414,10 +419,12 @@ module Dval = {
     | DResult(Belt.Result.t<t, t>)
     | DBytes(bytes)
 
-  /// Gets the Dark runtime type from a runtime value
+  @ocaml.doc("Gets the Dark runtime type from a runtime value")
   let rec toType = (dv: t): DType.t => {
-    // CLEANUP this function is derived from the backend version, but avoid blowing
-    // everything up I've kept it on the current types.
+    // CLEANUP this function is derived from the backend version, but to avoid
```

Comment:
Yeah, this is in the middle of a transition. We used to have non-polymorphic types. Now we've added polymorphic types, but we don't really use them, so this converts them into basically non-polymorphic types again, which is what most things and the clients expect.